site/source/tutorials/008_animations/demo/finished/zombies/src/widgets/Zombies.ts

```diff
@@ -153,7 +153,8 @@ export class Zombies extends WidgetBase {
 		return hearts;
 	}
 
-	private _onZombieLegsPlaybackRateChange(value: string) {
+	private _onZombieLegsPlaybackRateChange(event: Event) {
```

Comment:
The interface is `value: string` with the latest `Slider` version, see https://github.com/dojo/widgets/blob/master/src/common/interfaces.d.ts#L45

Reply:
@agubler  Yep, please abort this pull request, the embarrassment is killing me

Reply:
@DannyHellbane okay!